src/crud/webhook.py

```
@@ -0,0 +1,111 @@
+    Args:
+        db: Database session
+        workspace_name: Name of the workspace (optional)
+
+    Returns:
+        List of webhook endpoints
+    """
+    # Verify workspace exists
+    await get_workspace(db, workspace_name)
+
+    stmt = select(models.WebhookEndpoint).where(
+        models.WebhookEndpoint.workspace_name == workspace_name
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)